src/components/forms/SelectFormField.js

```diff
@@ -6,10 +6,18 @@ import FormErrorMessage from "./FormErrorMessage";
 import { Colors, Typography } from "../../constants/styles";
 import FontForgeIcon from "../common/FontForgeIcon";
 
-const SelectFormField = ({ name, leftText, rightText }) => {
+const SelectFormField = ({
+  name,
+  leftText,
+  rightText,
+  defaultOption,
+  calculateDependentValueWhenRightChecked,
+}) => {
   const { setFieldValue, errors, touched } = useFormikContext();
-  const [isLeftChecked, setLeftChecked] = useState(false);
-  const [isRightChecked, setRightChecked] = useState(false);
+  const [isLeftChecked, setLeftChecked] = useState(!!defaultOption);
```

Comment:
Dość nieintucyjny jest ten kod. Trzeba się dobrze wczytać, żeby zrozumieć o co chodzi. Może zrobić tylko jeden stan, który jest booleanem lub nullem. Przy lewym komponencie sprawdzać czy state === true, a przy prawym state === false, jeśli będzie null to żadna odpowiedź nie będzie zaznaczona.